### DISCUSSION TITLE
mdsf vs Prettier

### DISCUSSION THREAD

**Discussion by ccoVeille** [hougesen/mdsf#337] [General]
I found this old post by looking at something else

https://prettier.io/blog/2017/11/07/1.8.0.html

Doesn't it mean prettier is able to format the Markdown fenced blocks like you do?

So said otherwise, doesn't prettier support http fenced blocks?

Prettier use remark-parse, there might be a remark plugin

**Comment by hougesen** [comment]
Yes, prettier is able to format markdown codeblocks (and markdown as a whole), but only languages that it has a [plugin](https://prettier.io/docs/en/plugins#official-plugins) for.

Sadly I was unable to find a plugin for `.http` files  :(

**Reply by ccoVeille** [reply]
Thanks it's clearer

I think you should add a section to README about the limited support prettier has.

**Reply by hougesen** [reply]
Prettier comes with support for a lot of languages by default (screenshot). The plugins are only used for custom languages. 

![image](https://github.com/hougesen/mdsf/assets/56034786/a48e23c3-9585-477c-be09-bf23bb533697)

**Reply by ccoVeille** [reply]
Yes, you should mention that while prettier supports a large number of language, markdown is only one of them, and the support of fenced block formatting is very limited.

Here you could add the link to Puerto markdown plugins, and say you support 30+ (fix this) language
